src/main/groovy/org/pih/warehouse/importer/CycleCountItemsExcelImporter.groovy

```diff
@@ -97,6 +100,8 @@ class CycleCountItemsExcelImporter extends AbstractExcelImporter implements Data
                     name: personMap[row.assignee]?.name,
                     label: personMap[row.assignee]?.name,
             ] : null
+            // excelimport only supports the Date type so we need to convert to Instant ourselves.
+            row.dateCounted = instantValueConverter.convertString(row.dateCounted as String)
```

Comment:
Unfortunate that we have to do this but it's a consequence of using a deprecated excel importer library. I'd love to find the time to switch to a newer library that supports Instant and LocalDate out of the box